### ISSUE TITLE
really fix `install_t` in containers

### ISSUE THREAD

**Issue by cgwalters** [containers/bootc#83]
I'm not sure if something changed here, or my code never actually worked but basically we currently fail to `bootc install` with unknown types.

**Comment by akdev1l** [comment]
I hit this bug while trying to create a bootc image based on Arch Linux 

It seems that it can be worked around by disabling SELinux

**Comment by akdev1l** [comment]
so we tried this on a fedora host with SELinux disabled and `--disable-selinux` flag.

This allowed us to skip the `install_t` issue when creating an bootc image from arch linux (they don't ship selinux policies so everything is unlabeled). However we ran into some problems.

We realized `matchpathcon` and `chcon` are both used even when selinux is disabled at the kernel level and `--disable-selinux` has been passed so the target installed system should not necessarily have labels (though I guess it is nice to have them in case the user wants to turn selinux on later, it seems the idea behind `--disable-selinux` is to support non-selinux enabled system)

We were able to patch this by simply overwriting the binaries, `matchpathcon` was hardcoded to output a specific selinux context and `chcon` was overwritten to be equal to `/bin/true` so both commands would succeed unconditionally. (we're trying not to touch `bootc` itself for now, just trying to make it work on a non-fedora system as proof of concept)

is this behaviour expected?
